desktop-widgets/configuredivecomputerdialog.cpp

```diff
@@ -344,10 +344,10 @@ void OstcFirmwareCheck::saveOstcFirmware(QNetworkReply *reply)
 	dialog->setCancelButton(0);
 	dialog->setAutoClose(true);
 	ConfigureDiveComputer *config = new ConfigureDiveComputer();
-	connect(config, SIGNAL(message(QString)), dialog, SLOT(setLabelText(QString)));
-	connect(config, SIGNAL(error(QString)), dialog, SLOT(setLabelText(QString)));
-	connect(config, SIGNAL(progress(int)), dialog, SLOT(setValue(int)));
-	connect(dialog, SIGNAL(finished(int)), config, SLOT(dc_close()));
+	connect(config, &ConfigureDiveComputer::message, dialog, &QProgressDialog::setLabelText);
+	connect(config, &ConfigureDiveComputer::error, dialog, &QProgressDialog::setLabelText);
+	connect(config, &ConfigureDiveComputer::progress, dialog, &QProgressDialog::setValue);
+// 	connect(dialog, &QProgressDialog::finished, config, &ConfigureDiveComputer::dc_close); TODO: this is wrong
```

Comment:
I don't like the idea of commenting this out with a TODO. Either leave the old code, or figure out what we want / need there. But removing something and replacing it with nothing plus a comment that there's work left to do... not so cool.